src/components/molecules/UserProfilePicture/UserProfilePicture.styles.ts

```diff
@@ -49,14 +49,15 @@ export const Reaction = styled.div<ReactionProps>`
 
 // --- Reaction Container
 export const Container = styled.div`
-  // @debt convert this to scss then use our z-index layer helper here
-  z-index: 11;
-
   height: 100%;
 
   background-position: center;
   background-size: cover;
 
+  .reaction-container {
```

Comment:
There will be 2 instances of this rendered currently, one on shouts, one on emoji reactions. Unless you’re also doing things to position this over the top of the avatar, I don’t believe this will work correctly

Reply:
It works as expected or what do you mean?

![image](https://user-images.githubusercontent.com/63194656/114531918-0251ca80-9c55-11eb-8bd8-99c33deb593a.png)

Reply:
Like in my screenshots on the original bug fix, when there are multiple reactions happening at the same time, the original ‘basic fix’ ended up having some of the avatars overlaying the emoji/etc; whereas in my latest fix, we ensure that they always stay on heir appropriate layers as described above.

Simplest way to check this is to get userWithId from useUser in Audience, then change the seat code to render that user in every seat.

Then send a reaction, it should be obvious of the layering is working correctly or not (would be good to share a screenshot here to show that)

Reply:
Actually, the specific thing I meant here, which ‘direction’ are the reactions meant to appear. I believe top/right. Yet in your screenshot here it’s showing more bottom/right.

If you render a background/border/similar on the reaction-container I expect you’ll find it’s sitting beneath the user avatar, rather than in top of it.